.eslintrc.json

```diff
@@ -11,7 +11,7 @@
           "error",
           {
             "enforceBuildableLibDependency": true,
-            "allow": [],
+            "allow": ["@monorepo/**"],
```

Comment:
Reference: 

https://stackoverflow.com/questions/74296342/nx-error-for-relative-imports-within-the-same-project

Reply:
after reading this it seems like the relative pathing is encouraged within a module / lib and references should be across.  Maybe this is showing that we aren't necessarily splitting up our libs right?  Can you double check the usage and see how much are cross module referencs and how many are within a module?

Reply:
I think this is also indicating that we should be pulling out as much as we can from app and putting it into our libs and passing the config in maybe?

Reply:
I've decided to move the `config.ts` file under `libs/expo/shared/utils/src/lib/config.ts` as the `libs` directory is already enabled for sharing code. 

Regarding the cross module references, it should be fine as long as we're sharing and enforcing the scope between the libraries and projects. Here's a more detailed understanding of it through nx [1](https://nx.dev/getting-started/tutorials/react-monorepo-tutorial#imposing-constraints-with-module-boundary-rules) and [2](https://nx.dev/recipes/enforce-module-boundaries/tag-multiple-dimensions), and the reasons for its benefits [here](https://nx.dev/concepts/decisions/code-ownership).

I'm hesitant to tag anything without understanding the relationship between the apps and libraries, but welcome a discussion to flesh out the details. Once we get that achieved, it should be easy to implement.